scripts/_terraform.sh

```diff
@@ -527,6 +529,11 @@ function _get_target_aws_account_name() {
     fi 
 }
 
+
+_make_workspace_name() {
+    echo "$1" | openssl dgst -sha1 -binary | xxd -p | cut -c1-8
```

Comment:
This is how we come up with the environment ID.
In that instance we call `whoami` from the local machine and pipe that into this.